main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/constants/databricks/EDatabriksClusterType.java

```diff
@@ -0,0 +1,37 @@
+// ============================================================================
+//
+// Copyright (C) 2006-2021 Talend Inc. - www.talend.com
+//
+// This source code is available under agreement available at
+// %InstallDIR%\features\org.talend.rcp.branding.%PRODUCTNAME%\%PRODUCTNAME%license.txt
+//
+// You should have received a copy of the agreement
+// along with this program; if not, write to Talend SA
+// 9 rue Pages 92150 Suresnes, France
+//
+// ============================================================================
+package org.talend.hadoop.distribution.constants.databricks;
+
+import org.talend.hadoop.distribution.i18n.Messages;
+public enum EDatabriksClusterType {

```

Comment:
```suggestion
public enum EDatabricksClusterType {

```
There is same typo on other already existing enums. Not mandatory to change them in this feature

Reply:
ah good catch i'll correct them so its done